plugins/topology/src/const.ts

```diff
@@ -0,0 +1,20 @@
+export const DEFAULT_NODE_PAD = 20;
```

Comment:
Differentiating the ENUm declaration standard and constant declaration standard helps to make code more readable

Reply:
sorry if I get it right all enums I need to update to PascalCase i.e UpperCamelCase and constants should be as it's here or ? 

https://www.typescriptlang.org/docs/handbook/enums.html#:~:text=Enums%20are%20one%20of%20the,numeric%20and%20string%2Dbased%20enums.